src/content-handlers/iiif/extensions/uv-openseadragon-extension/config/Config.ts

```diff
@@ -142,7 +142,9 @@ type OpenSeadragonCenterPanelContent = CenterPanelContent & {
   goHome: string;
   imageUnavailable: string;
   next: string;
+  nextImage: string;
```

Comment:
I wonder if we should remove the plain `next` / `previous` if they are no longer used. Doesn't hurt anything to keep them, and maybe they're actually still used somewhere, but if they're definitely not needed, it might be nice to clear out some redundancy while we're thinking of it.